packages/arb-token-bridge-ui/src/components/common/Notifications.tsx

```diff
@@ -42,6 +46,59 @@ function Notification({
   )
 }
 
+function ToggleTheme() {
+  const [theme, setTheme] = useLocalStorage<string>('theme')
+  const classicThemeKey = 'arbitrum-classic-theme'
+  const [themeInState, setThemeInState] = useState<string | null>(theme)
+  const isClassicTheme = themeInState === classicThemeKey
+
+  useEffect(() => {
+    setTheme(classicThemeKey)
+    setThemeInState(classicThemeKey)
+  }, [setTheme])
+
+  useEffect(() => {
+    const elem = document.getElementById('body-theme')
+    if (!elem) return
+    elem.className = theme ?? ''
+  }, [theme])
+
+  const handleToggleTheme = () => {
+    if (theme === classicThemeKey) {
+      setTheme('')
+      setThemeInState('')
+    } else {
+      setTheme(classicThemeKey)
+      setThemeInState(classicThemeKey)
+    }
+  }
+
+  return (
+    <>
+      {isClassicTheme && (
+        <Notification>
+          <a
+            href="https://youtu.be/uHgt8giw1LY"
```

Comment:
Why not official video? https://www.youtube.com/watch?v=dQw4w9WgXcQ

Reply:
ads

Reply:
yea got 10 sec ads